2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_setup.py:_flush():70] Current SDK version is 0.19.11
2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_setup.py:_flush():70] Configure stats pid to 3808009
2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_setup.py:_flush():70] Loading settings from /home/aiotlab/.config/wandb/settings
2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_setup.py:_flush():70] Loading settings from /mnt/disk1/aiotlab/namth/EEGFoundationModel/CBraMod/wandb/settings
2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_setup.py:_flush():70] Loading settings from environment variables
2025-09-13 01:00:18,183 INFO    MainThread:3808009 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /mnt/disk1/aiotlab/namth/EEGFoundationModel/CBraMod/wandb/run-20250913_010018-xrb4hqse/logs/debug.log
2025-09-13 01:00:18,184 INFO    MainThread:3808009 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /mnt/disk1/aiotlab/namth/EEGFoundationModel/CBraMod/wandb/run-20250913_010018-xrb4hqse/logs/debug-internal.log
2025-09-13 01:00:18,184 INFO    MainThread:3808009 [wandb_init.py:init():852] calling init triggers
2025-09-13 01:00:18,184 INFO    MainThread:3808009 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'seed': 3407, 'cuda': 0, 'epochs': 50, 'batch_size': 64, 'lr': 0.0003, 'weight_decay': 0.05, 'optimizer': 'AdamW', 'clip_value': 1, 'dropout': 0.1, 'classifier': 'all_patch_reps', 'downstream_dataset': 'PEARL', 'datasets_dir': '/mnt/disk1/aiotlab/namth/EEGFoundationModel/datasets/pearl', 'num_of_classes': 2, 'model_dir': './data/wjq/models_weights/Big/BigFaced', 'num_workers': 16, 'label_smoothing': 0.1, 'multi_lr': False, 'frozen': False, 'use_pretrained_weights': True, 'foundation_dir': 'pretrained_weights/pretrained_weights.pth', '_wandb': {}}
2025-09-13 01:00:18,184 INFO    MainThread:3808009 [wandb_init.py:init():893] starting backend
2025-09-13 01:00:18,184 INFO    MainThread:3808009 [wandb_init.py:init():897] sending inform_init request
2025-09-13 01:00:18,191 INFO    MainThread:3808009 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-09-13 01:00:18,191 INFO    MainThread:3808009 [wandb_init.py:init():907] backend started and connected
2025-09-13 01:00:18,194 INFO    MainThread:3808009 [wandb_init.py:init():1005] updated telemetry
2025-09-13 01:00:18,203 INFO    MainThread:3808009 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-09-13 01:00:18,971 INFO    MainThread:3808009 [wandb_init.py:init():1104] starting run threads in backend
2025-09-13 01:00:19,319 WARNING MainThread:3808009 [wandb_init.py:init():1681] [no run ID] interrupted
Traceback (most recent call last):
  File "/home/aiotlab/miniconda3/lib/python3.13/site-packages/wandb/sdk/wandb_init.py", line 1677, in init
    return wi.init(run_settings, run_config, run_printer)
           ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/site-packages/wandb/sdk/wandb_init.py", line 1149, in init
    run._on_start()
    ~~~~~~~~~~~~~^^
  File "/home/aiotlab/miniconda3/lib/python3.13/site-packages/wandb/sdk/wandb_run.py", line 2602, in _on_start
    self._backend.interface.publish_python_packages(working_set())
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/site-packages/wandb/sdk/interface/interface.py", line 345, in publish_python_packages
    for pkg in working_set:
               ^^^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/site-packages/wandb/util.py", line 1941, in working_set
    yield InstalledDistribution(key=d.metadata["Name"], version=d.version)
                                                                ^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/importlib/metadata/__init__.py", line 486, in version
    return self.metadata['Version']
           ^^^^^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/importlib/metadata/__init__.py", line 471, in metadata
    return _adapters.Message(email.message_from_string(text))
                             ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/__init__.py", line 37, in message_from_string
    return Parser(*args, **kws).parsestr(s)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/parser.py", line 64, in parsestr
    return self.parse(StringIO(text), headersonly=headersonly)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/parser.py", line 53, in parse
    feedparser.feed(data)
    ~~~~~~~~~~~~~~~^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/feedparser.py", line 176, in feed
    self._call_parse()
    ~~~~~~~~~~~~~~~~^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/feedparser.py", line 180, in _call_parse
    self._parse()
    ~~~~~~~~~~~^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/feedparser.py", line 240, in _parsegen
    self._parse_headers(headers)
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/aiotlab/miniconda3/lib/python3.13/email/feedparser.py", line 489, in _parse_headers
    self._cur.set_raw(*self.policy.header_source_parse(lastvalue))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-09-13 01:00:19,978 INFO    MsgRouterThr:3808009 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 0 handles.
